app/pages/Accounts/AccountReport/util.ts

```diff
@@ -41,6 +51,13 @@ function calculateShieldedBalanceChange(
     transaction: TransferTransaction,
     isOutgoing: boolean
 ): string {
+    if (
+        [TransactionStatus.Failed, TransactionStatus.Rejected].includes(
+            transaction.status
+        )
+    ) {
+        return '0';
+    }
```

Comment:
I ændringen før denne, er der forskel på "rejected" og "failed" ift. om transaction cost bliver inkluderet. Hvordan kan det være, at det ikke gør sig gældende her?

Reply:
Failed transactions are those, which are added to the chain, but "failed", so the user still pay the cost, but the transfer doesn't happen. Rejected transactions do not make it on chain, so they have no effect.

That is why they are handled different on the public balance, but the cost does not affect the shielded balance.